ng2-material/components/data_table/data_table.ts

```diff
@@ -0,0 +1,199 @@
+import {Component, Input, Output, Inject, forwardRef, ElementRef, AfterContentInit, EventEmitter} from 'angular2/core';
+import {Observable} from 'rxjs/Observable';
+import {Observer} from 'rxjs/Observer';
+import 'rxjs/add/operator/filter';
+import 'rxjs/add/operator/share';
+import {MdCheckbox} from '../checkbox/checkbox';
+
+/**
+ * @name mdDataTable
```

Comment:
We don't parse the docs like the angular/material repo does.  I think this markdown documentation would be a great fit for being put in `examples/components/data_table/readme.md` file.